src/js/components/scenarios.js

```diff
@@ -0,0 +1,30 @@
+import { newText, newMultimedia, newContainer } from './helpers';
+
+const createImages = (dataImages) => {
+  const arrayImages = dataImages.map((element, number) => {
+    const imagesScenarios = newMultimedia('img', element, ['scenarios__image']);
+    const textScenarios = newText('p', [''], ['scenarios__text']);
+    textScenarios.innerHTML = element.text;
```

Comment:
Shouldn't the `element.text` be added in the previous line, where the `['']` are?

Reply:
It was a suggestion in order to avoid an error.

Reply:
You are getting an error because you are passing an array as second parameter to the `newText` function. It should be a string, like `newText('p', element.text, ['scenarios__text']);`

Reply:
Done Add the text.element parameter, in the empty array space

Reply:
Please remove the line 7 because you are already adding text to the `<p>` through the `newText` function